terraform/modules/task-definitions/publishing-api/main.tf

```diff
@@ -16,7 +16,7 @@ provider "aws" {
 }
 
 locals {
-  service_name = "publishing-api"
+  app_name = "publishing-api"
```

Comment:
Not something to worry about in this PR but I feel like the semantics of service_name vs app_name vs mesh_subdomain etc. are getting pretty confusing, especially now that we've subtly changed the meaning of service_name by adding the -web suffix to the main service.

Maybe we could refactor later to pass in app_name and app_name_suffix (or something) and compute service_name - I don't know.

Reply:
This config is a result of the publishing-api-web service tasks being reachable via `publishing-api.mesh.govuk-internal.digital` rather than `publishing-api-web...` which doesn't seem valuable, I'll remove that logic in a subsequent PR.